company_website/static/main_page/scripts/project-highlight.js

```diff
@@ -0,0 +1,132 @@
+projectHighlightOverlay = $(".project-highlight")[0];
+projects = $(".projects");
+projectBoxes = $(".project-box")
+screenMdSize = 991
+blurrablePageSections = [
+    // ".main-screen",
+    // ".container",
+    // ".testimonials-container",
+    // ".services-container",
+    // ".industry-focus-container",
+    // ".portfolio-container",
+    // ".clutch-container",
+    // ".ready-to-work-with-us",
+    // ".map",
+    // ".footer-section",
```

Comment:
Is this needs to be commented out or it can be deleted? Because I think that this is not used at all

Reply:
Done.